src/array.rs

```diff
@@ -75,6 +75,7 @@ use crate::types::Element;
 ///     array![[8., 15.], [12., 23.]]
 /// );
 /// ```
+#[derive(Debug)]
```

Comment:
This adds `PhatromData,PhantomData)` to its debug print, which I don't like.
So could you please use `pyobject_native_type_fmt!(PyArray<T, D>, T, D);` instead?

Reply:
Done.